consensus/polybft/state.go

```diff
@@ -602,29 +559,21 @@ func (s *State) insertBundles(bundles []*BundleProof) error {
 	})
 }
 
-// getBundles gets bundles that are not executed
-func (s *State) getBundles(stateSyncExecutionIndex, maxNumberOfBundles uint64) ([]*BundleProof, error) {
-	var bundles []*BundleProof
+// getStateSyncProof gets state sync proof that are not executed
+func (s *State) getStateSyncProof(stateSyncExecutionIndex uint64) (*types.StateSyncProof, error) {
```

Comment:
Nitpick: Maybe rename this `stateSyncExecutionIndex` to `stateSyncID`?

Reply:
Done in https://github.com/0xPolygon/polygon-edge/pull/1013/commits/8a39e5377aeef0fe0d04fc62daecbd3b01bf3242